python/pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 markers =
     smoke_test: mark a test as a build verification testing.
-addopts = -k "not test_redis_modules.py"
+addopts = -k "not test_redis_modules.py and not test_json.py"
```

Comment:
instead of adding this for each new module we add, can you check if we can move all module tests into a folder (tests/modules) and skip the whole folder by default?